charts/opentelemetry-collector/values.yaml

```diff
@@ -82,8 +82,12 @@ configMap:
   create: true
   # Specifies an existing ConfigMap to be mounted to the pod
   # The ConfigMap MUST include the collector configuration via a key named 'relay' or the collector will not start.
+  # If bundling this external ConfigMap via a wrapper chart, the name of the
+  # ConfigMap file must be in the form '<existingName_value>.yaml' to allow
+  # the chart to calculate a checksum annotation.
```

Comment:
What happens if the form is wrong, will the chart fail to install?

Reply:
It doesn't look like it's possible to feed template contents to Helm except by reading a file, so I [brought back](https://github.com/open-telemetry/opentelemetry-helm-charts/pull/1302/commits/15445ca23b07ca198b20f988b24f6c60253107e1) the config key to set the file path.

Reply:
Closing the loop on your original question, we're protecting the user from a failed install by avoiding setting the new option in the default `values.yaml`. A user will have to explicitly enable this key to trigger checksumming of a custom template file.